src/main/java/me/lokka30/levelledmobs/customdrops/CustomDropsHandler.java

```diff
@@ -33,15 +35,7 @@
 import java.math.BigDecimal;
 import java.math.RoundingMode;
 import java.sql.Timestamp;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
we shouldn't use star imports - [here's how to disable it](https://stackoverflow.com/a/3589885) :)